openedx/custom/db/fields.py

```diff
@@ -0,0 +1,79 @@
+from django.core import exceptions
+from django.utils.text import capfirst
+from multiselectfield import MultiSelectField
+from multiselectfield.db.fields import MSFList
+from multiselectfield.utils import string_type, get_max_length
+
+from openedx.custom.forms.fields import MultiSelectWithOtherFormField
+
+
+class MultiSelectWithOtherField(MultiSelectField):
+
+    def __init__(self, other_max_length=None, *args, **kwargs):
```

Comment:
Let's please log proper error message if `other_max_length` is not provided just like dJango provides in case we didn't pass max_length to CharFields. 

> marketplace.MSModel.my_field1: (fields.E120) CharFields must define a 'max_length' attribute.